### PR TITLE
WIP: Open Data Blog Post 

### PR DESCRIPTION
**Pull request by noamoss**
Blogpost for proofreading.

### CODE REVIEW

**Comment by RachelBaram** [comment]
@noamoss I get the feeling that this was written in Hebrew first and then translated and there are important points not coming through - literally lost in translation. If this is the case, can you share the Hebrew version with me. It will be easier for me to ensure that the edits reflect your original intentions.

**Comment by noamoss** [comment]
Actually, I just had a list of some key points and Hebrew, which I used as
a base structure.
If you prefer just to mark parts with unclear meanings I can try to
simplify and/or elaborate.

*Noam Castel*,

Account Manager
Gizra Internet Solutions Ltd.

*Gizra*
http://www.gizra.com  | +972-3-3731222 | noam@gizra.com <orit@gizra.com>

P Think before you print.

File: app/_posts/2016-11-26-open-data.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+There are two basic questions to address:
+
+ 1. **What should be considered Public Data?**
+Well, it is a pretty large and abstract question, and in a way, also semi-philosophic. For the sake of efficiency in this short post, let’s agree we refer to any kind of information or raw data being held and managed by public authorities and has the potential to influence decisions that we, as citizens, make in our life.
+
+ 2. **What is considered opened Public Data?**
+I love this question because it is a great case study in which civic society preceded and set the definitions to be used later by state organizations as standards.
+In the US, [The original principles of Open Governmental Data](https://public.resource.org/8_principles.html) were defined almost a decade ago (2007) in Sebastopol, California, paving the way for the [American Open Government Initiative](https://www.whitehouse.gov/open):
+
+	 1. Complete - all of the data is available, not just a part
```
Comment (RachelBaram) [inline app/_posts/2016-11-26-open-data.md]: This original list is written in a very precise and clear way. I would consider just using a block quote instead.

File: app/_posts/2016-11-26-open-data.md [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+	 6. Non-discriminatory (Make it available to all, without any demand to register).
+	 7. Non-proprietary (Neither the content nor the format should be limiting users).
+	 8. License-free (except for some reasonable privacy, security and privilege restrictions. These words might make you yawnת but actually it is one of the most tumultuous arenas of struggles.)
+
+These roots of the open data definition, like any other open source project, are [dynamic](http://opendefinition.org/participate/), [forked](http://opendatahandbook.org/guide/en/what-is-open-data/), and [evolving](http://opendefinition.org/od/2.1/en/). And so are the measurement and ranking tools being used in the field - compare for example the [World Wide Web Foundation's Barometer](http://opendatabarometer.org/) with the [Open Knowledge's Global Open Data Index](http://index.okfn.org/)).
+
+
+## OK, so why should we care ?
+
+Open source projects have common threads with open data initiatives, and therefore we might learn a lesson or two out of the similarities. Which similarities?
```
Comment (RachelBaram) [inline app/_posts/2016-11-26-open-data.md]: I read about the similarities but what are the lessons that you refer to?

File: app/_posts/2016-11-26-open-data.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+* In both fields we care, a lot, about the [data / code] end users. Most of the time, they are at the center of our discussion.
+* [The userstory](https://en.wikipedia.org/wiki/User_story) is a NeverEnding story: we can’t, and shouldn’t, assume nor predict how the [data / code] will be reused later and embrace dynamic.
+
+{% include thumbnail.html image_path="assets/images/posts/open-data-blogpost/neverending_storyjpg" %}
+Atrribution: By Michael Kleinhenz from Bonn, Germany (dscf1619.jpg) [CC BY-SA 2.0 (http://creativecommons.org/licenses/by-sa/2.0)], via Wikimedia Commons
+
+* No [data / code / change ] is hidden. Here is the (whole and original) source, use it as you want.
+* Don’t be afraid, we can always reverse changes and follow history.
+
+To sum it all, we can simplify the differences betweeh open data/source actors in a very visual way: How do you like your clay/data/source code?
```
Comment (RachelBaram) [inline app/_posts/2016-11-26-open-data.md]: I don't understand what you are trying to say here. Is this the lesson we should learn?

File: app/_posts/2016-11-26-open-data.md [hunk 4]
```diff
@@ -0,0 +1,66 @@
...
+	 3. Timely - make it available ASAP, since time influences the data’s usability and value
+	 4. Accessible - consider the means of distribution to enable as many as possible to use the data).
+	 5. Machine Processable (...make it usable, not only readable).
+	 6. Non-discriminatory (Make it available to all, without any demand to register).
+	 7. Non-proprietary (Neither the content nor the format should be limiting users).
+	 8. License-free (except for some reasonable privacy, security and privilege restrictions. These words might make you yawnת but actually it is one of the most tumultuous arenas of struggles.)
+
+These roots of the open data definition, like any other open source project, are [dynamic](http://opendefinition.org/participate/), [forked](http://opendatahandbook.org/guide/en/what-is-open-data/), and [evolving](http://opendefinition.org/od/2.1/en/). And so are the measurement and ranking tools being used in the field - compare for example the [World Wide Web Foundation's Barometer](http://opendatabarometer.org/) with the [Open Knowledge's Global Open Data Index](http://index.okfn.org/)).
```
Comment (RachelBaram) [inline app/_posts/2016-11-26-open-data.md]: What will the reader see if they compare the WWW foundation's barometer with the Open Knowledge's Global Open Data Index?  Its great that you have a lot of links but the reader should not be "forced" to leave your blog to understand the points.

File: app/_posts/2016-11-26-open-data.md [hunk 5]
```diff
@@ -0,0 +1,66 @@
...
+description: "One of the fascinating, and less noticeable, evolutions of the open source movement is found in the changing arena of public data."
+---
+
+{% include setup %}
+
+What is Open - Public - Data?
+-------------
+More and more people are able to define Open Source, mostly in terms of its technological and economic aspects. It has become really easy to see the tremendous effects of the open source perspective in the development processes (i.e. collaborative programming) and in day-to-day accessories (at least those using a linux based mobile phone). There has also been a fascinating, and less noticeable, evolution of the open source thought in the field of public data.
+
+We are talking about evolving the relationships between citizens, social activists and governments, in an era where reproducing the raw material of decisions costs as much as (...or less than) a [rubber duck](http://www.ebay.com/sch/items/?_nkw=rubber+ducks&_sacat=&_ex_kw=&_mPrRngCbx=1&_udlo=&_udhi=&_sop=12&_fpos=&_fspt=1&_sadis=&LH_CAds=&rmvSB=true).
```
Comment (RachelBaram) [inline app/_posts/2016-11-26-open-data.md]: Why is this an important point? There is a "raw material" thread throughout the blog you are trying to use that is getting lost. If I don't get it, most of your readers won't either. We are Gizra... no fluff. Its very important to be clear as possible to what you mean.  ;)  This is a fascinating topic!

**Comment by IshaDakota** [comment]
@noamoss A general comment after a brief read: I think this has the potential to be a fascinating piece. @RachelBaram is providing good guidance on the details - I might suggest that you lead with something more personal. 

In our brief conversation I learned about your rich experience in opendata/sunlight initiatives. Why are you so interested in these and why are they important?... then you can turn to the reader and explain why they should think it's interesting and important.

Not a lot - just 3-5 sentence to set it up.

**Comment by noamoss** [comment]
Thanks Adam, it helps. I will send a revised draft soon.

**Comment by RachelBaram** [comment]
@IshaDakota Noam wrote a new version [here](https://docs.google.com/document/d/1v5F0nY2XyPMUQO99RFq54rzNw-KQBugXoZL-zXGtgOM/edit#heading=h.jcavry8qlrw2). He is having an issue with local installment and hence was not able to update it here. He is super busy this week and will not get to solving the technical issues on his computer. Can you help me here? Its important we get a blog out this week. Thanks!

**Comment by IshaDakota** [comment]
closed in favor of #541

**Pull request closed without merging**